### ISSUE TITLE
Widgets taking more than one designated space

### ISSUE THREAD

**Issue by bartoszhernas** [macoscope/GrandCentralBoard#23]
Hey, I have just stumbled on the project after reading about it on your blog :)
I love the idea and in the spare time, I will probably write some widgets for it. 

What do you think to allow each widget to take more than 1 dimension? 
Eg. I can see that time widget needs 1 tile, but twitter widget would work better taking up two tiles (vertically of course). Same goes for eg. photos view which would be better to be two tiles horizontally (but one tile here is also fine). Charts are also the perfect example. 

Let me know what you think!

**Comment by ochococo** [comment]
Hey! 

Thank you.

This dillema is as old as the project itself. Personally I would be in favor but I would like to discuss this issue a little bit more. There is one problem. If you have 6 uniform widgets - none of them will block the other.

Consider this situation:
![widgets](https://cloud.githubusercontent.com/assets/3382607/13700306/7461e3b0-e780-11e5-9d3f-005883c1b4e1.png)

You cannot add another vertical widget. You're stuck. And that's why the Widget is 640x540 now.

Thanks for sharing your idea with us!

**Comment by bartoszhernas** [comment]
Yes, it's stuck, but it's up to the user. There will be always limitations. If you add 6 current-size widgets, then you cannot also add more. I understand your point.

Just mentioned the idea, in the later stage of the project it will give much more flexibility to widgets developers.
Nevertheless, it's the perfect project for me to display GAnalytics and Stripe data which I always wanted. 

Will share it later, thanks a lot for creating this!

**Comment by ochococo** [comment]
Yeah, you're right.

It's almost certain we will add this feature.

**Comment by joeljfischer** [comment]
It would be nice for widgets to define multiple possible layouts. So a widget could define a 1x1 layout, 1x2, 2x1, 3x1, etc. if they wanted to.

**Comment by ochococo** [comment]
Great idea.. Thanks!

**Comment by ochococo** [comment]
But I'm reluctant to add complexity without some significant gain... This Issue is out from `Core 2.0` for now.
